src/vector/mod.rs

```diff
@@ -22,6 +22,7 @@ mod geo_to_gdal;
 mod geometry;
 mod layer;
 mod ops;
+pub mod sql;
```

Comment:
Sorry, GitHub is not letting me comment on your original comment now, but I actually had a similar thought. I changed it instead to just make the `sql` module public, that way users can choose to namespace as `sql::ResultSet` if they desire. How does this sound? (Still open to feedback, I just changed it because of the asynchronicity of these reviews.)

Reply:
sounds good to me